userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md

```diff
@@ -0,0 +1,135 @@
+Flux will then sync the new files and the profile will be applied to your cluster.
+
+:::info
+When installing a profile via its URL (ie when using the `--profile-url` flag)
+remember to check where the profile's `profile.yaml` file is located within
+the profile's source repository.
+
+Once discovered, you can set the relative path to this file using the `--profile-path` flag.
+:::
+
+## Further configurations
+
+You can pass further arguments to `pctl install` for more control over the format
+and destination of your PR. For example, the following command will:
+- generate the profile manifests in the `ethel` directory within my GitOps repo
+- set the objects to be deployed in the `two-sugars-please` namespace in my cluster
+- commit to a branch called `add-aunt-ethels-profile`
+- push that branch to the `jammy-dodger` remote
+- open a PR against the `tea-time` branch in my target GitOps repo
+
+```bash
+pctl install \
+  --profile-url https://github.com/weaveworks/nginx-profile \
+  --create-pr \
+  --pr-repo drwho/thirteen \
+  --out ethel \
+  --namespace two-sugars-please \
+  --pr-branch add-aunt-ethels-profile \
+  --pr-remote jammy-dodger \
+  --pr-base tea-time
+```
+
+To see all flags available on this subcommand, see [the help](/docs/pctl/install).
+
+## Installing a profile which contains local resources
+
+A local resource is any profile artifact which refers to manifests stored within the profile
+repository. You can see [this page](/docs/author-docs/local-helm-chart) from the author docs as an example.
+
+In order to install profiles with local resources, you must provide `pctl` with information
+about your GitOps repo (very meta, I know).
+
+First, look up the ID of the `GitRepository` resource connected to your repo. (This is
+what Flux uses internally to keep things up to date between the repo and the cluster.)
+
+```bash
+# replace GITOPS_REPO with the name of your GitOps repository
+
+kubectl get gitrepositories.source.toolkit.fluxcd.io -A | awk '/GITOPS_REPO/ {print $1"/"$2}'
+```
+
+This should return something like:
+
+```bash
+# namespace/name
+flux-system/flux-system
+```
+
+Then add the following flag to your `install` command:
+
+```sh
+--git-repository <output from the command above>
+```
+
+### How do I know if the profile I want to install has local resources?
+
+To find this out you should navigate to the repository of the profile and look at the `profile.yaml`.
+In this file you will find a list of `artifacts`. Those which have local resources can be identified
+with one of the following keys:
+- `chart.path`
+- `kustomize.path`
+
+Take care to also check whether the list of artifacts includes any `profile.source` keys,
+as these may point to other profiles which themselves could contain local resources.
+
+Profile authors may also note in their documentation that their profile contains
+local resources, but this should not be relied upon.
```

Comment:
I wonder how much its worth being like "if your doing X you must provide the git repository" vs just always providing it? We have 3 artifact types (excluding nested profiles), 2/3 require this flag. In any non-basic profile I'd imagine that pretty quickly it will always be the case you will need to provide this.

Reply:
should we make it a required flag then?

Reply:
mhmm I'd rather just always document it as being needed, even if cheekly there are edgescases where its not? wdyt? 

If you install a profile that needs it you do get quite a nice error message `failed to build artifact: in case of local resources, the flux gitrepository object's details must be provided`

Reply:
okay i have changed it to recommend that it always be set